.travis.yml

```diff
@@ -1,8 +1,10 @@
 dist: trusty
+addons:	
+  chrome: stable
 before_install:
   - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.21.1
   - export PATH="$HOME/.yarn/bin:$PATH"
-  - google-chrome-stable --headless --disable-gpu --remote-debugging-port=9222 --no-sandbox --disable-dev-shm-usage http://localhost &
+  - google-chrome-stable --headless --disable-gpu --remote-debugging-port=9222 http://localhost &
```

Comment:
I'm not entirely sure that Selenium webdriver even uses this running instance of chrome. I actually wonder if the tests would run without this line

Reply:
I can test removing that line! also what do you mean by their chrome plugin? would that let us specify a version of chrome?

Reply:
`chrome-stable` is their plugin for providing chrome, and it's annoying because you can't specify an exact version. I put a link to it in that old PR that you linked to

Reply:
Oh, are you referring to [`this addon`](https://github.com/CMSgov/design-system/pull/623/files/c818dc4b26ba7adde0703c2cfec4e0b4e9d0169a#diff-354f30a63fb0907d4ad57269548329e3R3)? I tried removing that and the tests seem to run fine, is that preferable?

Reply:
Yeah, if that line isn't doing anything, let's get rid of it. It just confuses things.

Reply:
Sorry, I just meant this line `google-chrome-stable --headless`, not removing the add-on. We need the add-on so that chrome is installed

Reply:
Wait, sorry, you removed the addon and it still worked? If that's the case, don't take my word for it that it's necessary to keep around!

Reply:
What a great mystery Travis CI is 🔮

Reply:
No it doesn't work when I remove it, I just made a mistake haha